ext/neural_snow/ModelTools.jl

```diff
@@ -428,4 +785,121 @@ function trainmodel!(
         cb()
     end
 end
+
+
+"""
+   save_predictive_model_weights(path, model)
+
+A helper/utility function for writing the weights of the PREDICTIVE COMPONENT
+of a model made with `make_model()` or `make_model_paper()` to a human-parseable text file.
+
+This only saves the weights of the predictive component of the network, and only
+if the predictive component is a chain of named `Dense` layers, in the same manner
+as the paper by Charbonneau et. al. (2025). For custom models or alternative constructions
+we refer the reader to the Flux standard Saving and Loading page: https://fluxml.ai/Flux.jl/stable/guide/saving/
+This provides parameter files that can be readily used with other coding languages and is human readable,
+as well as working independent of the Flux package version.
+
+# Arguments
+- `path`::String: The filepath for which to save the text file.
+- `model`: the model for which to save the predictive parameters.
+"""
```

Comment:
Please, add an header with the version number to the output file (so that we can define compatibility across versions of ClimaLand).